src/util/aws/api-gateway.js

```diff
@@ -16,9 +19,14 @@ export function exportStage (restApiId, stageName) {
   .get('body')
 }
 
-export function deploy (id, env) {
+export function deploy (id, env, attempts = 1) {
   const apiGateway = new AWS.APIGateway()
   return apiGateway.createDeployment({restApiId: id, stageName: env, variables: { functionAlias: env }}).promise()
+  .catch({ code: 'TooManyRequestsException' }, async ({ retryable, retryDelay }) => {
+    if (!retryable && attempts > DEPLOY_ATTEMPT_MAX) { throw new Error('Amazon limit hit') }
+    await Promise.delay(Math.ceil(retryDelay * 1000))
```

Comment:
I'm assuming it's seconds (values range from 10-40), I wasn't able to find good docs on this error object. Wasn't sure how Bluebird would handle rounding so figured I would just do that here